pkg/cosign/git/github/github.go

```diff
@@ -140,3 +138,29 @@ func (g *Gh) PutSecret(ctx context.Context, ref string, pf cosign.PassFunc) erro
 func (g *Gh) GetSecret(ctx context.Context, ref string, key string) (string, error) {
 	return "", nil
 }
+
+func encryptSecretWithPublicKey(publicKey *github.PublicKey, secretName string, secretValue []byte) (*github.EncryptedSecret, error) {
+	decodedPubKey, err := base64.StdEncoding.DecodeString(publicKey.GetKey())
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode public key: %w", err)
+	}
+	var peersPubKey [32]byte
```

Comment:
What's the advantage of this extra variable? Maybe a comment explaining why we can't just use `decodedPubKey`.

Reply:
i need to copy the 32 first and then pass that as a reference to the function, cannot use that straight